Components/Rating/rating.js

```diff
@@ -0,0 +1,112 @@
+let ratingStars = document.querySelectorAll(".rating-stars");
+let resetStarsBtn = document.querySelectorAll(".star-reset-btn");
+let allRatings = document.querySelectorAll('.rating-value');
+let productRatingStars = document.querySelectorAll('.product-rating-stars')
+let productRatingStats = document.querySelector("#product-stat");
+
+
+function productAverageRating(averageRating)
+{
+    // Ex - averageRating=3.5 
+    // 3    -   3.1, 3.2, 3.3, 3.4    
```

Comment:
Are you adding this comment as an example?
If as an example then okay, otherwise you can remove it.